src/tests/ie_test_utils/common_test_utils/test_constants.hpp

```diff
@@ -9,7 +9,7 @@ namespace CommonTestUtils {
 const char DEVICE_AUTO[] = "AUTO";
 const char DEVICE_CPU[] = "CPU";
 const char DEVICE_GNA[] = "GNA";
-const char DEVICE_GPU[] = "GPU";
+extern const char* DEVICE_GPU; //definition: test_common.cpp
```

Comment:
I'd suggest to keep that part as is and try to set property for GPU plugin with default device id: https://github.com/openvinotoolkit/openvino/blob/master/src/inference/include/openvino/runtime/properties.hpp#L530

Reply:
@vladimir-paramuzov Currently, there are two ways to select device. One is changing test_constants `DEVICE_GPU="GPU.1"` (option1) and the other one is `core.set_property("GPU", {{"DEVICE_ID", "1"}})` (option2).

IMO, option1 is better way. I've tried option2 but it doesn't cover as many testcase as option1, because some testcases are using GetCPPPluginByName without creating ov::Core(). Moreover, option1 is better for the perspective of consistency because implementation of -d option in benchmark_app also using device_name(e.g. "GPU.1") directly.

When using option1, all test-related global variable's initialization order have to be delayed, but I believe option1 is better way.